packages/app-next/src/App.tsx

```diff
@@ -125,6 +126,7 @@ const app = createApp({
     userSettingsPlugin,
     homePlugin,
     appVisualizerPlugin,
+    devToolsPlugin,
```

Comment:
This shouldn't be needed, works without this addition as well right?

Reply:
If I take it out I get: "ReferenceError: devToolsPlugin is not defined" in the UI

Reply:
Have you added the plugin to the `package.json` dependencies and installed it?

Reply:
It actually was there before I even started work on this: https://github.com/backstage/backstage/blob/56d3373c8262e947747b7dfe399d9a5870ff775a/packages/app-next/package.json#L37

Reply:
I rebased and removed this just to see if it worked without and it does 🚀

Reply:
**Note:** I removed the linguist import as well